RFS/src/main/java/com/rfs/common/RestClient.java

```diff
@@ -34,6 +29,7 @@ public RestClient(ConnectionDetails connectionDetails) {
             .baseUrl(connectionDetails.url)
             .headers(h -> {
                 h.add("Content-Type", "application/json");
+                h.add("User-Agent", "RfsWorker");
```

Comment:
Oh I was hoping we could pull that number from a variable or a commit sha.  `1.0` is useful when we've got an official version out, but depending on how the tool is deployed I think I'd want to be able to checkout a branch at a specific commit id to troubleshoot or know if something was fixed in a future revision.

This could be good to pull out as new task.